lte/gateway/fabfile.py

```diff
@@ -31,28 +31,7 @@
 
 sys.path.append('../../orc8r')
 import tools.fab.pkg as pkg
```

Comment:
`  # noqa: E402` can be used here to remove the warnings. using `sys.path.append('../../orc8r')` in the first place seems hacky, but I do not see a lightweight clean solution.

Reply:
@nstng @wolfseb I was always somewhat confused by this import. This is clearly just a hack because you can't do relative imports natively like that. But why are the fab tools located in `orc8r` anyway? There is no fabric scripts in the orc8r component and from what I can tell, everything in `orc8r/tools/fab` is used by the `lte` and `feg` components. Couldn't we at least move all the fab tools over to the `lte`?

Reply:
This will not solve all places, e.g, `feg/gateway/fabfile.py`.